parity/rpc_apis.rs

```diff
@@ -566,6 +570,13 @@ impl ApiSet {
 				public_list.insert(Api::Personal);
 				public_list
 			},
+			ApiSet::PubSub => vec![
```

Comment:
Small grumble: could be just a slice or a fixed array instead of a vec? Same for line 539.